From b83fd54584fabd5d24f6645b4a3cf345c9d2020d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 17 Jun 2017 16:11:59 -0700
Subject: [PATCH 1/3] ttcp: Add printf format string

Fixes compiler warnings when format security is enabled

| ../../../linux-atm-2.5.2/src/test/ttcp.c:666:21: error: format not a string literal and no format arguments [-Werror=format-security]
|      fprintf(stderr, Usage);
|                      ^~~~~

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/test/ttcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test/ttcp.c b/src/test/ttcp.c
index acb9185..337cee5 100644
--- a/src/test/ttcp.c
+++ b/src/test/ttcp.c
@@ -663,7 +663,7 @@ int no_check = 0;
     exit(0);
 
   usage:
-    fprintf(stderr, Usage);
+    fprintf(stderr, "%s", Usage);
     exit(1);
 }
 
-- 
2.13.1

